deploy/openshift/clusterrole.yaml

```diff
@@ -79,6 +79,16 @@ rules:
   - get
   - list
   - watch
+- apiGroups: ["authentication.k8s.io"]
+  resources:
+  - tokenreviews
+  verbs:
+  - create
+- apiGroups: ["authorization.k8s.io"]
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - create
```

Comment:
I don't understand why we need this. Once we get the token we shouldn't have to do anything extra with it unless I am missing something.